Dockerfile.test

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.12.0-alpine3.9
+FROM golang:1.13-alpine
```

Comment:
I think we'll want this at 1.15 - but I've just pinged @BradleyBoutcher, and he may fix it elsewhere